### PR TITLE
Implement EIP-5920

### PR DESCRIPTION
**Pull request by Pandapip1**
## Changes

- Adds support for [EIP-5920](https://eips.ethereum.org/EIPS/eip-5920)

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [x] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a fix or a feature that would cause existing functionality not to work as expected)
- [ ] Documentation update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional or API changes)
- [ ] Build-related changes
- [ ] Other: _description_ 

#### Requires testing

- [x] Yes
- [ ] No

#### If yes, did you write tests?

- [x] Yes
- [ ] No

#### Notes on testing

Have not written tests. I will be writing tests, and this PR will remain a draft until I have written them.

### CODE REVIEW

**Comment by Pandapip1** [comment]
I'm waiting for the checks to run to see what code styling things need fixing and what build errors I've introduced.

**Review by LukaszRozmej — CHANGES_REQUESTED** [review]
Looks good, lacks any tests though

**Comment by Pandapip1** [comment]
> Looks good, lacks any tests though

Will definitely write those. Somehow this PR fell off my to do list.

**Comment by Pandapip1** [comment]
Added tests, but I'm not sure the tests work.

**Comment by Scooletz** [comment]
Converting to `draft` for a book keeping purposes.

**Comment by Pandapip1** [comment]
@Scooletz mind running the tests?

**Comment by Scooletz** [comment]
@LukaszRozmej could you help with running the tests?

File: src/Nethermind/Nethermind.Specs/Forks/17_Cancun.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ protected Cancun()
         {
             Name = "Cancun";
             IsEip1153Enabled = true;
-            IsEip4844Enabled = true;
+            IsEip5920Enabled = true;
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Specs/Forks/17_Cancun.cs:17]: Eip5920 is not yet confirmed to be in cancun, 4844 is.

**Comment by Pandapip1** [comment]
There are some unrelated issues related to EIP-4844 that are causing some tasks to fail.

**Comment by Pandapip1** [comment]
@LukaszRomej mind re-approving the workflows?

**Comment by Pandapip1** [comment]
I'm going to need some help. I have no idea how to `Execute` in the context of a specific account.

**Pull request closed without merging**